test/test_build.py

```diff
@@ -113,11 +113,20 @@ def build_images(shared_tmpdir, build_container, request, force_aws_upload):
     password = "password"
     kargs = "systemd.journald.forward_to_console=1"
 
+    container_ref = tc.container_ref
+
+    # We cannot use localhost as we need to access the registry from both
+    # the host system and the bootc-image-builder container.
+    default_ip = testutil.get_ip_from_default_route()
+    local_registry = f"{default_ip}:5000"
```

Comment:
(nitpick) we have a "testutil:get_free_port()" helper (which is sadly inherently racy) I think would be slightly nicer in case 5000 is already taken - or will that break things if we don't use 5000 (it seems we pass the port everywhere?)

Reply:
The port number itself is not important but the idea is to share the same registry to save all the signed images locally. I will add a global `registry_port` variable, not sure if that's the best way to do it though

Reply:
I think the "correct" way would be a pytest fixture with the scope set to session.

Reply:
Used the `get_free_port()` function as requested